skimage/measure/_structural_similarity.py

```diff
@@ -74,7 +74,7 @@ def compare_ssim(X, Y, win_size=None, gradient=False,
        structural similarity. IEEE Transactions on Image Processing,
        13, 600-612.
        https://ece.uwaterloo.ca/~z70wang/publications/ssim.pdf,
-       DOI:10.1.1.11.2477
+       DOI:10.1109/TIP.2003.819861
```

Comment:
The old one is still valid - http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.11.2477